Microsoft.Toolkit.Uwp.UI.Controls.Layout/GridSplitter/GridSplitter.xaml

```diff
@@ -27,14 +28,28 @@
         <Setter Property="MinHeight" Value="16" />
         <Setter Property="Background" Value="{ThemeResource SystemControlHighlightChromeHighBrush}" />
         <Setter Property="GripperForeground" Value="{ThemeResource SystemControlForegroundAltHighBrush}" />
+        <Setter Property="HorizontalContentAlignment" Value="Center" />
+        <Setter Property="VerticalContentAlignment" Value="Center" />
+        <Setter Property="ContentTemplate">
+            <Setter.Value>
+                <DataTemplate>
+                    <TextBlock AutomationProperties.AccessibilityView="Raw"
+                               FontFamily="Segoe MDL2 Assets"
```

Comment:
I think your on Windows 11, so this works fine, eh? I know there's oddity now with MDL2 on Win10/11 or with the new symbol font or something. @zateutsch do you remember what that was again? I think it was [this issue](https://github.com/zateutsch/ML-Playground/issues/4) but I didn't see the commit linked which resolved it, as I don't remember what you had before.

Reply:
If I remember correctly, they recommended binding to the ThemeResource FontFamily as to avoid problems across 10 and 11. The new font family does contain a few symbols that the old doesn't, but for most use cases it should be ok.

Reply:
`SymbolFonts` font resource removed, and using `FontFamily="{ThemeResource SymbolThemeFontFamily}"`  (suggested `ThemeResource FontFamily` leads to runtime error).